File: quan.cs/Test/Config/Common/ItemConstant.cs
namespace Test.Config.Common
{
    /// <summary>
    /// 自动生成
    /// </summary>
    public class ItemConstant 
    {
        /// <summary>
        /// 常量1
        /// </summary>
        public static int Constant1 => ConstantConfig.GetByKey("constant1").ItemId;

        /// <summary>
        /// 常量2
        /// </summary>
        public static int Constant2 => ConstantConfig.GetByKey("constant2").ItemId;

        /// <summary>
        /// 常量3
        /// </summary>
        public static int Constant3 => ConstantConfig.GetByKey("constant3").ItemId;
    }
}

File: quan.lua/src/test/config/common/ItemConstant.lua
local ConstantConfig = require("test.config.common.ConstantConfig")

---
---@author 代码自动生成，请勿手动修改
local ItemConstant = {}

---
---常量1
---@return int
function ItemConstant.constant1()
    return ConstantConfig.getByKey("constant1").itemId
end

---
---常量3
---@return int
function ItemConstant.constant2()
    return ConstantConfig.getByKey("constant2").itemId
end

return ItemConstant